public/render.js

```diff
@@ -58,13 +58,13 @@ let render = {
         <h2>${item.name}</h2>
         <em>Type: ${item.type}</em>
         <p>${item.description}</p>
-        </div>
-        <div class="actions">
         <p>Posted by: ${item.postedBy}</p>
         <p>${(((item.acceptedBy) && (item.type !== 'Loan')) ? item.status+' by: '+item.acceptedBy : '')}</p>
         <p>${(((item.acceptedBy !== null) && (item.type === 'Loan') && (item.status !== 'Borrow'))? item.status+' by: '+item.acceptedBy : '')}</p>
+        </div>
+        <div class="actions">
         ${(((item.status === 'Borrow' || item.status === 'Claim' || item.status === 'Make Offer') && (item.postedBy !== STORE.currentUser)) ? `<button type="button" data-item-id="${item._id}" data-item-type="${item.type}" class="action-btn btn ${item.status.replace(' ','-')}">${item.status}</button>` : '')}
-        ${((item.status === 'On Loan' || item.status === 'Claimed' || item.status === 'Purchased') ? `<div class="js-status-tag"> ${item.status} </div>`: '')}
+        ${((item.status === 'On Loan' || item.status === 'Claimed' || item.status === 'Purchased') ? `<button class="js-status-tag" disabled> ${item.status} </button>`: '')}
```

Comment:
Nice!  A button was better than the div as it allows for an easier time positioning since we can reuse the .btn class